src/utils/mergejson.py

```diff
@@ -71,5 +74,10 @@
             unicode('utt2spk', 'utf-8'):dic[unicode('utt2spk', 'utf-8')]}
     
     # ensure "ensure_ascii=False", which is a bug
-    jsonstring = json.dumps({'utts': new_dic}, indent=4, ensure_ascii=False, sort_keys=True).encode('utf_8')
-    print(jsonstring)
+    if not args.output_json:
```

Comment:
How about the following?
Then, we can still use a pipe.
```
    if args.output_json:
	with codecs.open(args.output_json, "w", encoding='utf-8') as json_file:
            json.dump({'utts': new_dic}, json_file, indent=4, ensure_ascii=False, sort_keys=True, encoding="utf-8")
    else:
        sys.stdout = codecs.getwriter('utf8')(sys.stdout)
        json.dump({'utts': new_dic}, sys.stdout, indent=4, ensure_ascii=False, sort_keys=True, encoding="utf-8")
```

Also, make sure to import sys and codec in the header
```
import argparse
import codecs
import json
import logging
import sys
```